### PR TITLE
No longer attempting to format arguments when none are present

### PR DESCRIPTION
**Pull request by PawelPlesniak**
The hard coded `rprint` in line 101 prints the options when running `boot`. This is hard coded for now, will be imported from the controller driver in the next release. Issue created - https://github.com/DUNE-DAQ/drunc/issues/148
The corrections in line 164-166 remove the `'NoneType' object has no attribute 'get'` issues observed today.

### CODE REVIEW

File: src/drunc/controller/interface/shell_utils.py [hunk 1]
```diff
@@ -97,6 +97,8 @@ def controller_setup(ctx, controller_address):
 
         if ret.flag == ResponseFlag.EXECUTED_SUCCESSFULLY:
             ctx.info('You are in control.')
+            from rich import print as rprint
```
Comment (alessandrothea) [inline src/drunc/controller/interface/shell_utils.py:100]: Why not simply `from rich import print` module wise?
This should really be the default everywhere.

**Review by alessandrothea — CHANGES_REQUESTED** [review]

File: src/drunc/controller/interface/shell_utils.py (hunk 1 above)
Comment (alessandrothea) [reply src/drunc/controller/interface/shell_utils.py:100]: Looks good otherwise

**Review by TiagoTAlves — APPROVED** [review]

File: src/drunc/controller/interface/shell_utils.py [hunk 2]
```diff
@@ -97,6 +97,8 @@ def controller_setup(ctx, controller_address):
 
         if ret.flag == ResponseFlag.EXECUTED_SUCCESSFULLY:
             ctx.info('You are in control.')
+            from rich import print
```
Comment (alessandrothea) [inline src/drunc/controller/interface/shell_utils.py:100]: Move this to the top of the file

**Review by alessandrothea — CHANGES_REQUESTED** [review]

File: src/drunc/controller/interface/shell_utils.py (hunk 2 above)
Comment (TiagoTAlves) [reply src/drunc/controller/interface/shell_utils.py]: Done

**Review by alessandrothea — APPROVED** [review]
LGTM

**Pull request merged**